### PR TITLE
Feature/110/interview stepper

### PR DESCRIPTION
**Pull request by Dmitruk89**
1. change styles of modal header and buttons.
2. add appoint interview stepper
screen:
Снимок экрана 2021-11-18 в 15 08 16

### CODE REVIEW

File: src/app/components/appoint-interview-popup/appoint-interview-popup.component.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<app-modal-window [title]="title">
+  <div class="card-popup_container">
+    <div class="section">
+      <mat-stepper #stepper>
+        <mat-step [stepControl]="firstFormGroup" errorMessage="Date is required.">
+          <form [formGroup]="firstFormGroup">
+            <ng-template matStepLabel>Choose the date</ng-template>
+            
+            <mat-form-field appearance="fill" class="example-form-field">
+              <mat-label>Choose a date</mat-label>
```
Comment (mhasilau) [inline src/app/components/appoint-interview-popup/appoint-interview-popup.component.html:10]: pls add translate at here and below

File: src/app/components/candidate-table/candidate-table/candidate-table.component.html [hunk 2]
```diff
@@ -39,7 +39,7 @@
 </mat-form-field>
 
 
-<div class="mat-elevation-z8">
+<div class="mat-elevation-z8 candidate-table">
     <table mat-table [dataSource]="dataSource" matSort>
         <!-- Checkbox Column -->
```
Comment (mhasilau) [inline src/app/components/candidate-table/candidate-table/candidate-table.component.html:44]: pls delete comments

File: src/app/components/appoint-interview-popup/appoint-interview-popup.component.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  font-size: 16px;
+}
+
+.close-btn_container{
+  text-align: end;
+}
+
+.stepper-btn_container{
+  text-align: end;
+}
```
Comment (mhasilau) [inline src/app/components/appoint-interview-popup/appoint-interview-popup.component.scss:39]: pls add empty line in all docs

File: src/app/components/appoint-interview-popup/appoint-interview-popup.component.html (hunk 1 above)
Comment (vstrabko) [reply src/app/components/appoint-interview-popup/appoint-interview-popup.component.html:10]: +1

**Review by vstrabko — CHANGES_REQUESTED** [review]

File: src/app/components/appoint-interview-popup/appoint-interview-popup.component.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+              <input matInput (dateInput)="addEvent($event)" [matDatepicker]="datepicker" formControlName="firstCtrl" required>
```
Comment (ozhurova) [inline src/app/components/appoint-interview-popup/appoint-interview-popup.component.html:11]: Can you  add only a calendar or input is  required?

File: src/app/components/appoint-interview-popup/appoint-interview-popup.component.ts [hunk 5]
```diff
@@ -0,0 +1,79 @@
+import { InterviewerService } from './../../services/interviewer.service';
+import { Interviewer } from 'src/app/models/interviewer.model';
+import { Component, Input, Output, EventEmitter, OnInit } from '@angular/core';
+import { Candidate } from 'src/app/models/candidate.model';
+import { ModalWindowService } from '../modal-window/modal-window.service';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { STEPPER_GLOBAL_OPTIONS } from '@angular/cdk/stepper';
+import { MatDatepickerInputEvent } from '@angular/material/datepicker';
+
+interface Interval {
```
Comment (ozhurova) [inline src/app/components/appoint-interview-popup/appoint-interview-popup.component.ts:10]: We have interface.ts

**Pull request merged**